src/components/side-panel/common/dataset-info.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 Uber Technologies, Inc.
```

Comment:
why is this recognized as new file? instead of rename

Reply:
fixed